apps/journeys-admin/src/components/JourneyVisitorsList/FilterDrawer/ClearAllButton/ClearAllButton.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import { Meta, Story } from '@storybook/react'
+import { ComponentProps } from 'react'
+import { journeysAdminConfig } from '../../../../libs/storybook'
+import { ClearAllButton } from './ClearAllButton'
+
+const ClearAllButtonStory = {
+  ...journeysAdminConfig,
+  component: ClearAllButton,
+  title: 'Journeys-Admin/JourneyVisitorsList/FilterDrawer/ClearAllButton',
+  parameters: {
+    ...journeysAdminConfig.parameters,
+    layout: 'fullscreen'
+  }
+}
+
+const Template: Story<ComponentProps<typeof ClearAllButton>> = () => {
+  return (
+    <>
+      <ClearAllButton />
+    </>
+  )
+}
```

Comment:
We don't need any custom actions in the template, so we won't need to declare the return statement and can be just a standard arrow function
```suggestion
const Template: Story<ComponentProps<typeof ClearAllButton>> = () => ( <ClearAllButton /> )
```